client/engine/engine.go

```diff
@@ -453,6 +521,24 @@ func (e *Engine) attemptProgress(objective protocols.Objective) (outgoing Object
 	return
 }
 
+func (e Engine) registerVirtualChannelWithManagers(vfo virtualfund.Objective) error {
+	prefund := vfo.V.PreFundState()
+	startingBalance := big.NewInt(0)
+	// TODO: Assumes one asset for now
+	startingBalance.Set(prefund.Outcome[0].Allocations[0].Amount)
+
+	switch vfo.MyRole {
+	case 0:
+		return e.pm.Register(vfo.V.Id, startingBalance)
+	case 2:
+		return e.rm.Register(vfo.V.Id, prefund.Participants[0], startingBalance)
+	default:
+		// The intermediary does not need to use the payment or receipt manager
```

Comment:
Nit: intermediaries

Reply:
Done in [0490cd7](https://github.com/statechannels/go-nitro/pull/836/commits/0490cd7480d27e8c67ad81e7f18c9cb7eb093bd5)